Dockerfile

```diff
@@ -44,7 +45,7 @@ FROM checkmarx/kics:alpine as kics
 FROM trufflesecurity/trufflehog:latest as trufflehog
 FROM jdkato/vale:latest as vale
 FROM lycheeverse/lychee:latest-alpine as lychee
-FROM ghcr.io/terraform-linters/tflint:v0.51.0 as tflint
+FROM ghcr.io/terraform-linters/tflint:v${TFLINT_VERSION} as tflint
```

Comment:
I'm not sure, but maybe I'll have to include the v inside the env var. And anyways, if someone wants to use the variable to point to a different tag, that restricts to tags starting with v

Reply:
I'd like to keep SEMVER values for versions ^^